File: C++/Grade.cpp
#include<iostream>
using namespace std;
int main()
{
    float s1,s2,s3,per;
    cout<<"\nEnter The Marks of three Subjects : ";
    cout<<"\n Physics : ";
    cin>>s1;
    cout<<" Chemistry : ";
    cin>>s2;
    cout<<" Maths : ";
    cin>>s3;
    per=(s1+s2+s3)/300*100;
    cout<<"\nPercentage = "<<per<<"%\n\n";
    if(per>90&&per<=100)
    {
        cout<<"Grade : 'O' ";
    }
    else if(per>80&&per<=90)
    {
        cout<<"Grade : 'A+' ";
    }
    else if(per>70&&per<=80)
    {
        cout<<"Grade : 'A' ";
    }
    else if(per>60&&per<=70)
    {
        cout<<"Grade : 'B+' ";
    }
    else if(per>50&&per<=60)
    {
        cout<<"Grade : 'B' ";
    }
    else
    {
        cout<<"Grade : 'P' ";
    }
    cout<<"\n";
    return 0;
}

File: Python/Grade.py
#WAP to input Five Subject Marks and calculate total marks , Percentage and Grade.

phy=int(input("Enter Marks of Physics : "))
che=int(input("Enter Marks of Chemistry : "))
mat=int(input("Enter Marks of Maths : "))
eng=int(input("Enter Marks of English : "))
hin=int(input("Enter Marks of Hindi : "))

total=phy+che+mat+eng+hin
per=total/5

if (per>90 and per<100):
    grade="O"
elif (per>80 and per<=90):
    grade="A+"
elif (per>70 and per<=80):
    grade="A"
elif (per>60 and per<=70):
    grade="B+"
elif (per>50 and per<=60):
    grade="B"
elif (per>40 and per<=50):
    grade="C"
else:
    grade="F"

print("Total Marks = ",total)
print("Percentage = ",per)
print("Grade = ",grade)